src/Analyzers/CSharp/CodeFixes/ConvertToRecord/ConvertToRecordEngine.cs

```diff
@@ -786,10 +785,8 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
                                     tokens = tokens.Replace(tokens[0], tokens[0].WithoutLeadingTrivia());
 
                                     if (index == summaryContent.Count - 1 &&
-                                        tokens.Length >= 2 &&
-                                        tokens[^1].IsKind(SyntaxKind.XmlTextLiteralToken) &&
-                                        tokens[^1].Text.GetFirstNonWhitespaceIndexInString() == -1 &&
-                                        tokens[^2].IsKind(SyntaxKind.XmlTextLiteralNewLineToken))
+                                        tokens is [.., (kind: SyntaxKind.XmlTextLiteralNewLineToken), (kind: SyntaxKind.XmlTextLiteralToken)] &&
+                                        tokens[^1].Text.GetFirstNonWhitespaceIndexInString() == -1)
```

Comment:
Could capture `tokens[^1]` in the pattern

Reply:
done.  thanks!